docs/functions.rst

```diff
@@ -82,3 +82,35 @@ HIPBLASLT_COMPUTE_F32   HIPBLASLT_R_32F       HIPBLASLT_R_16F HIPBLASLT_R_16F
 HIPBLASLT_COMPUTE_F32   HIPBLASLT_R_32F       HIPBLASLT_R_16F HIPBLASLT_R_32F
 HIPBLASLT_COMPUTE_F32   HIPBLASLT_R_32F       HIPBLASLT_R_16B HIPBLASLT_R_16B
 ======================= =================== ============= ==============
+
```

Comment:
You miss adding hipblasLtMatrixTransformDesc_t in documentation

Reply:
@Serge45 what I means is that you should add hipblasLtMatrixTransformDesc_t into hipBLASLt Datatypes section in documentation.

Reply:
Thanks for the reminder! amended.